systemtest/src/test/java/io/strimzi/systemtest/KafkaST.java

```diff
@@ -1407,6 +1408,46 @@ void verifyVolumeNamesAndLabels(int kafkaReplicas, int diskCountPerReplica, int
         }
     }
 
+    @Test
+    void testPersistentStorageSize() throws Exception {
+        String diskSizeGi = "70Gi";
```

Comment:
why not using a couple of different storage sizes just to test that actually they will be set differently?

Reply:
Yes, that's a good point but question is. Is there any special storage size that's has to be tested? I can do this test @parametrized.

Reply:
I don't think so. My suggestion was more about testing that if you ask for 5Gi on volume 0 and 10Gi on volume 1 it doesn't turn in the other was around (10Gi on volume 0 and 5Gi on volume 1) so checking that operator is doing it in the right way. Using one size, it's something you cannot test.